### DISCUSSION TITLE
Brouter: despite of avoid freeways these are routed mainly

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#225] [Q&A] [answered]
Can someone please tell how to set Brouter offline options to strictly avoid freeways? 
Despite of setting option avoid freeways, motorbike internal profile routes mainly through freeways.

**Comment by devemux86** [comment]
@DolomitesTom

Can you post all BRouter options and some route examples e.g. from [BRouter website](https://brouter.de/brouter-web/)?
Then it will be easy to check the OpenStreetMap tags on these roads.

Also you probably mean avoid motorways?

**Reply by devemux86** [reply]
Also, better start with few avoid options, like only avoid motorways and then proceed with the others.

The more avoidance options are enabled, the fewer roads are available for use.

**Reply by DolomitesTom** [reply]
Yes I mean motorways

These are the settings and it takes all motorways it can find.

![Screenshot_20230625-134036_Cruiser](https://github.com/devemux86/cruiser/assets/121371629/42be9d61-9f8a-4b63-a905-1024919d1223)

[Test.zip](https://github.com/devemux86/cruiser/files/11859575/Test.zip)

**Reply by DolomitesTom** [reply]
The option "Höchstgeschwindigkeitsstraßen vermeiden" in km/h has somehow no effect at all or I don't know what it means

**Reply by devemux86** [reply]
> The option "Höchstgeschwindigkeitsstraßen vermeiden" in km/h has somehow no effect at all or I don't know what it means

It is the vmax parameter in [BRouter](https://brouter.de/brouter-web/).
It should avoid roads above that speed.

I will rename it in the next update or we probably do not need the vmax and totalweight,
when we already have such detailed avoid options per road type?

**Reply by DolomitesTom** [reply]
Thanks and don't know, maybe still keep it for now. The problem was that it was set to an external profile. I only thought it was set to the internal. With the internal profile or works to not use motorways.

**Comment by devemux86** [comment]
Loading the Cruiser profile in [BRouter website](https://brouter.de/brouter-web/) we have the following results:
- Motorways on
- Motorways off

Press the "**Analyse route**" button at the right to see all road types in the route.
There are no motorway tags. Or do you mean another road tag?

![Screenshot 2023-06-25 at 15-16-46 BRouter web client](https://github.com/devemux86/cruiser/assets/3484020/a12900e7-239f-4a08-9bf9-90a97112817f)
![Screenshot 2023-06-25 at 15-17-23 BRouter web client](https://github.com/devemux86/cruiser/assets/3484020/bb0c1004-c88f-4152-a7cd-10b43aa2ac6f)

**Reply by DolomitesTom** [reply, marked as answer]
With the internal profile: driving-cruiser it works, thanks. I think it was a local problem.

**Reply by devemux86** [reply]
The internal profile driving-cruiser contains many more detailed avoid options.

If you use an external profile, some BRouter options are common and should work:
- avoid_toll
- avoid_unpaved
- avoid_motorways
- vmax
- totalweight

[BRoute website](https://brouter.de/brouter-web/) has the excellent "Analysis" and "Data" tools that help us see what is included in a route.
